### ISSUE TITLE
Slow sur main de liberté / tempête de lames

### ISSUE THREAD

**Issue by Kaderharicot** [way-of-elendil/3.3.5#1609]
**Description**
En ayant le buff main de liberté d'un paladin, ou en étant en train de faire une tempête de lames en guerrier, nous ne devrions pas pouvoir être slow, or sur woe ça arrive régulièrement

**Étapes de reproduction**
Fournissez les informations nécessaires pour reproduire le problème. Par exemple
Classes: 
Guerrier & paladin
Etape de reproduction, faire un duel avec un mage, spammer avec ce mage les frostbolt niveau 1 sur quelqu'un qui a main de liberté ou tempête de lame, il arrive régulièrement que cette personne soit slow

**Comportement attendu**
Ne pas être slow tant que les auras main de liberté ou tempête de lames sont actives

**Captures d’écran / Vidéos**

**Ressources additionnelles**
C'est le principe des spells, rien à link

**Comment by Ferro-woe** [comment]
> frostbolt niveau 1

Tous les niveaux du sort [éclair de givre](https://woehead.way-of-elendil.fr/?search=%C3%A9clair+de+givre#abilities) sont à prendre en compte, le _niveau 1_ n'est pas une obligation, le _niveau 16_ a les mêmes effets pour ce qui nous occupe. La différence est son temps d'incantation et ses dégâts.

**Comment by Kaderharicot** [comment]
Effectivement, j'ai pas précisé, la niveau 1 c'est juste pour pouvoir spammer pour le test

**Comment by sithis2a** [comment]
Ce bug a déjà été évoqué et "refusé" [ici](https://github.com/way-of-elendil/3.3.5/issues/1364).
Cependant la vidéo partagée date du 18 Juin 2009 et la 3.3.5 (notre version du jeu) date du 22 Juin 2010, il se peut qu'il y ait des différences. De plus sur Classic (je refuse le "c pa blizzlike" d'avance), ce bug n'est pas présent. 

Le moyen le plus connu de reproduire est de lancer la Bladestorm ou la HoF lorsqu'une Frostbolt est en cours de vol (c'est pour ça que c'est plus souvent reproduit avec un Fb r1, ou alors avec les images miroirs du Mage).

**Comment by MissZay** [comment]
> Une hypothèse est que la Frostbolt mettant un debuff de reduc heal avec le talent [Permafrost](https://www.wowhead.com/wotlk/spell=12571/permafrost), il se peut que le snare reste présent avec le debuff dans cas cas précis.

Même si c'est le cas utiliser un sort d'immunité est censé enlever tous les debuffs concernés, qu'ils aient un effet secondaire ou pas, or ici c'est assez fréquent d'utiliser une main de lib et de garder le debuff
Les seules exceptions qu'il devrait y avoir sont certaines mécaniques importantes de boss qui ont des tag pour ignorer les immunes, que ce soit immune au slow ou aux dégâts (souffle, glaçon et rafale sur sindra, limons sur putri, big bang/ascension sur algalon etc)

**Comment by sithis2a** [comment]
> > Une hypothèse est que la Frostbolt mettant un debuff de reduc heal avec le talent [Permafrost](https://www.wowhead.com/wotlk/spell=12571/permafrost), il se peut que le snare reste présent avec le debuff dans cas cas précis.
> 
> Même si c'est le cas utiliser un sort d'immunité est censé enlever tous les debuffs concernés, qu'ils aient un effet secondaire ou pas, or ici c'est assez fréquent d'utiliser une main de lib et de garder le debuff Les seules exceptions qu'il devrait y avoir sont certaines mécaniques importantes de boss qui ont des tag pour ignorer les immunes, que ce soit immune au slow ou aux dégâts (souffle, glaçon et rafale sur sindra, limons sur putri, big bang/ascension sur algalon etc)

Ai-je dit que c'était normal ? J'ai juste donné un potentiel point de départ pour le debug et au passage un tip pour se prendre moins (voir pas) de snare sous lib.
D'ailleurs pour le souffle de sindra, la lib ne vire pas le debuff mais retire bien le snare le temps de la lib.

**Comment by MissZay** [comment]
> D'ailleurs pour le souffle de sindra, la lib ne vire pas le debuff mais retire bien le snare le temps de la lib.
C'est exactement ce que j'ai dit par rapport au fait que certaines mécaniques de boss ignorent le fonctionnement "normal" de certains sorts
Pour le reste du message je visais pas particulièrement ce que tu disais mais juste je notais qu'ils y a pleins d'occurrences de ce bug sur le serveur
